napari/_vispy/layers/points.py

```diff
@@ -43,41 +47,41 @@ def _on_data_change(self):
             # always pass one invisible point to avoid issues
             data = np.zeros((1, self.layer._slice_input.ndisplay))
             size = np.zeros(1)
-            edge_color = np.array([[0.0, 0.0, 0.0, 1.0]], dtype=np.float32)
+            border_color = np.array([[0.0, 0.0, 0.0, 1.0]], dtype=np.float32)
             face_color = np.array([[1.0, 1.0, 1.0, 1.0]], dtype=np.float32)
-            edge_width = np.zeros(1)
+            border_width = np.zeros(1)
             symbol = ['o']
         else:
             data = self.layer._view_data
             size = self.layer._view_size
-            edge_color = self.layer._view_edge_color
+            border_color = self.layer._view_border_color
             face_color = self.layer._view_face_color
-            edge_width = self.layer._view_edge_width
+            border_width = self.layer._view_border_width
             symbol = [str(x) for x in self.layer._view_symbol]
 
         set_data = self.node._subvisuals[0].set_data
 
         # use only last dimension to scale point sizes, see #5582
         scale = self.layer.scale[-1]
 
-        if self.layer.edge_width_is_relative:
-            edge_kw = {
+        if self.layer.border_width_is_relative:
+            border_kw = {
                 'edge_width': None,
-                'edge_width_rel': edge_width,
+                'edge_width_rel': border_width,
             }
         else:
-            edge_kw = {
-                'edge_width': edge_width * scale,
+            border_kw = {
+                'edge_width': border_width * scale,
                 'edge_width_rel': None,
             }
 
         set_data(
             data[:, ::-1],
             size=size * scale,
             symbol=symbol,
-            edge_color=edge_color,
+            edge_color=border_color,
```

Comment:
@Czaki yeah I think it's a bit annoying but not enough to warrant waiting for the upstream PR :woman_shrugging:

Reply:
It could be done on napari side.

```
class OurMarker(Marker):
   def set_data(..., border_color, ...):
       super().set_data(..., edge_color=border_color, ...).
```
But I'm not sure if it worth.

Reply:
Ahh I see. I think that's odd tbh. If I was adding a new subvisual, I might go check out the `vispy` docs, and I would try to use `edge_*` without realizing it's been overridden. But I don't have strong feelings here!

Reply:
Maybe a comment to clarify?

Reply:
@psobolewskiPhD added comment 😄